firmware/quadruna/CRIT/src/tasks.c

```diff
@@ -377,6 +379,15 @@ bool (*heartbeatFaultGetters[HEARTBEAT_BOARD_COUNT])() = {
     [CRIT_HEARTBEAT_BOARD] = NULL
 };
 
+static const CritShdnConfig crit_shdn_pin_config = {
+    2,
+    inertia_sen_pin,
+    shdn_sen_pin,
+};
```

Comment:
suggestion: 
In newer versions of C you can use "designated initializers", which lets you annotate initializing structs.
```c
static const CritShdnConfig crit_shdn_pin_config = {
    .crit_num_nodes =2,
    .shdn_sen_ok_gpio = shdn_sen_pin,
    .inertia_sen_ok_gpio = inertia_sen_pin,
};
```
Helps you avoid bugs where you initialize things in the wrong order. Looks like there is actually an ordering bug rn, so this is always good practice in my opinion!